content/faq/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title : "Frequently Asked Questions"
+description: "Frequently asked questions about LocalStack, the services, licenses, and more."
+date: 2022-01-06T08:47:36+00:00
+lastmod: 2023-01-06T08:47:36+00:00
+draft: false
+images: []
+---
+
+## General
+
+- What is LocalStack?
+
+    LocalStack is a platform that simplifies and accelerates development of cloud applications. At its core, LocalStack provides a cloud service emulation layer that runs in a single container on your laptop or in your CI environment. With LocalStack, you can run your AWS applications or Lambdas entirely on your local machine without connecting to a remote cloud provider.
```

Comment:
I guess "AWS applications" is not wrong, but AWS services would cover things like running an SQS queue (service) which I wouldn't consider an application.